dotCMS/src/integration-test/java/com/dotcms/datagen/TestDataUtils.java

```diff
@@ -936,28 +941,43 @@ public static Relationship relateContentTypes(final ContentType parentContentTyp
     }
 
     public static Contentlet getFileAssetContent(Boolean persist, long languageId) {
+        return getFileAssetContent(persist, languageId, TestFile.JPG);
+    }
+
+    public static Contentlet getFileAssetSVGContent(Boolean persist, long languageId) {
+        return getFileAssetContent(persist, languageId, TestFile.SVG);
+    }
+
+    public static Contentlet getFileAssetContent(final Boolean persist, final long languageId, final TestFile testFile) {
 
         try {
-            Folder folder = new FolderDataGen().nextPersisted();
+            final Folder folder = new FolderDataGen().nextPersisted();
 
             //Test file
-            final String testImagePath = "com/dotmarketing/portlets/contentlet/business/test_files/test_image1.jpg";
+            final String testImagePath = testFile.filePath;
             return createFileAsset(testImagePath, folder, languageId, persist);
         } catch (Exception e) {
             throw new DotRuntimeException(e);
         }
     }
 
-    public static Contentlet getFileAssetSVGContent(Boolean persist, long languageId) {
+    public enum TestFile {
 
-        try {
-            final Folder folder = new FolderDataGen().nextPersisted();
+        JPG("com/dotmarketing/portlets/contentlet/business/test_files/test_image1.jpg"),
```

Comment:
Codacy found an issue: [The String literal "com/dotmarketing/portlets/contentlet/business/test_files/test_image1.jpg" appears 4 times in this file; the first occurrence is on line 966](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)